demos/icon-toggle.html

```diff
@@ -166,20 +137,6 @@ <h2>Additional Color Combinations</h2>
               </div>
               <div>Dark icon on background</div>
             </div>
-            <div id="custom-on-dark" class="demo-color-combo mdc-theme--dark">
```

Comment:
I think we actually want to keep this example, because it shows how to use custom colors. Just remove the `mdc-theme--dark` class.

Reply:
roger!